src/ui/Facet/Facet.ts

```diff
@@ -1528,6 +1528,35 @@ export class Facet extends Component {
     this.dependsOnManager = new DependsOnManager(facetInfo);
   }
 
+  private dependsOnUpdateParentDisplayValue() {
+    if (this.options.dependsOn) {
+      const masterFacetHTML = $$(this.root)
+        .findAll(`.CoveoFacet`)
+        .filter(HTMLFacet => {
+          const facet = <Facet>Facet.get(HTMLFacet);
+          return facet.options.id === this.options.dependsOn;
+        });
+      if (!masterFacetHTML.length) {
+        this.logger.warn(
+          `Unable to find a Facet with the id or field "${this.options.dependsOn}".`,
```

Comment:
Since we're actually looking for facets with a matching `id` option value, I'd rather say:

```
`Unable to find a facet with id "${this.options.dependsOn}".`,
```

For the second sentence, I personally find "master Facet" a little vague. Suggestion:

```
`The dependent facet cannot be updated.`
```